runtime/src/main/java/org/corfudb/runtime/collections/CorfuTable.java

```diff
@@ -761,36 +751,32 @@ public V merge(K key, V value, BiFunction<? super V, ? super V, ? extends V>
      */
     @SuppressWarnings("unchecked")
     protected void unmapSecondaryIndexes(K key, V value) {
+        if (value == null) {
+            log.trace("Attempting to build an index with a null value, skipping.");
```

Comment:
I think these can be warn level logs.
Trace is disabled by default on the clients.

Reply:
Yep.